core/src/main/java/org/apache/calcite/plan/volcano/VolcanoPlanner.java

```diff
@@ -276,9 +264,6 @@ public VolcanoPlanner(RelOptCostFactory costFactory, //
     super(costFactory == null ? VolcanoCost.FACTORY : costFactory, //
         externalContext);
     this.zeroCost = this.costFactory.makeZeroCost();
-    // If LOGGER is debug enabled, enable provenance information to be captured
-    this.provenanceMap = LOGGER.isDebugEnabled() ? new HashMap<>()
-        : Util.blackholeMap();
   }
```

Comment:
I guess you are able to saw such a change ~20 minutes ago in this PR. That's because I was resolving some conflicts and the PR was not ready at that time. 

Now in master branch the changes should be already corrected.

Reply:
Ok, thanks for your clarification~~